crates/agglayer/tests/snapshots/config__config_display.snap

```diff
@@ -0,0 +1,41 @@
+---
+source: crates/agglayer/tests/config.rs
+expression: result
+---
+[full-node-rpcs]
+
+[log]
+level = "info"
+outputs = []
+format = "pretty"
+
+[rpc]
+port = 9090
+host = "0.0.0.0"
+
+[outbound.rpc.settle]
+max_retries = 3
+retry_interval = 7
+confirmations = 1
+
+[l1]
+ChainID = 1337
+NodeURL = "http://zkevm-mock-l1-network:8545/"
+RollupManagerContract = "0xb7f8bc63bbcad18155201308c8f3540b07f84f5e"
+
+[auth.Local]
+private_keys = []
+
+[telemetry]
+PrometheusAddr = "0.0.0.0:3000"
+
+[epoch.TimeClock]
+EpochDuration = 5
+
+[shutdown]
+runtime_timeout = 5
```

Comment:
Quite a few of these are still not kebab case. I see a mixture of pascal, snake, kebab. Am I missing something? As far as I can tell, this is not related to the backwards compat test.